src/net/sourceforge/kolmafia/swingui/panel/AdventureSelectPanel.java

```diff
@@ -535,12 +535,12 @@ public static JPanel getAdventureSummary(final String property) {
     private final String property;
     private final CardLayout resultCards;
     private final JPanel resultPanel;
-    private final JComboBox resultSelect;
+    private final JComboBox<?> resultSelect;
```

Comment:
This instance looks like it's only populated above in this file with a JComboBox<String>, right?